From 962f69972db40bafe6ff41f2151fa4150bdda37d Mon Sep 17 00:00:00 2001
From: Samuel Verschelde <stormi@laposte.net>
Date: Thu, 13 Aug 2020 15:26:43 +0200
Subject: [PATCH 03/11] Add TrueNAS multipath config

This was a patch added to the sm RPM git repo before we had this
forked git repo for sm in the xcp-ng github organisation.
---
 multipath/multipath.conf | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/multipath/multipath.conf b/multipath/multipath.conf
index ab6cfe1..9a7205a 100644
--- a/multipath/multipath.conf
+++ b/multipath/multipath.conf
@@ -92,4 +92,11 @@ devices {
 		prio		        alua
 		uid_attribute		ID_SERIAL
 	}
+	device {
+		vendor			"TrueNAS"
+		product			"iSCSI Disk"
+		hardware_handler	"1 alua"
+		path_grouping_policy	group_by_prio
+		prio			alua
+	}
 }
-- 
2.36.0

